.github/workflows/release.yaml

```diff
@@ -0,0 +1,83 @@
+on:
+  push:
+    branches:
+      - master
+name: release-please
+jobs:
+  # Create and maintain release PRs:
+  release-please-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - id: release-pr
+        uses: GoogleCloudPlatform/release-please-action@v2
+        with:
+          path: packages/gcf-utils
+          token: ${{ secrets.FORKING_TOKEN }}
+          fork: true
+          release-type: node
+          package-name: gcf-utils
+          command: release-pr
+      # Forking token does not have permission to add label, so we use
+      # this script to do so:
+      - id: label
+        if: ${{ steps.release-pr.outputs.pr }}
+        uses: actions/github-script@v3    
+        with:
+            github-token: ${{secrets.GITHUB_TOKEN}}
+            script: |
+              const [owner, repo] = process.env.GITHUB_REPOSITORY.split('/');
+              await github.issues.addLabels({
```

Comment:
We've already seen frequent cases where Octokit just kinda breaks their API.  This ... makes me nervous.  How do we know what version of Octokit this uses?  Is it pinned inside of `actions/githubb-script`?

Reply:
@JustinBeckwith Octokit has much more frequently broken their types than their actual API, which I've rarely needed to make an update for. GitHub script is managed by the actions organization itself, and I'm fairly confident in its stability.

Reply:
It's also crossed my mind that perhaps I could deliver an easy to configure testing harness for GitHub script; then we could use it as a hammer for small tasks like adding labels, but have the confidence of tests.